en/manual/get-started/launch-stride.md

```diff
@@ -22,7 +22,7 @@ The Stride Visual Studio extension lets you you [edit shaders directly from Visu
 
 You can install multiple versions of Stride and launch them from the Stride Launcher.
 
-![Manage versions of Stride](media/stride-launcher-various-versions.png)
+![Manage versions of Stride](media/stride-launcher-various-versions.webp)
 
 You might need to use an older version of Stride to work with old projects. Newer versions of Stride might contain changes that require old projects to be upgraded.
 
```

Comment:
Could this new image `media/stride-launcher-start-button.webp` replace also the one here in the line 47?

```
![Start button](media/stride-launcher-start-button.png)
```